script/openqa-enqueue-asset-cleanup

```diff
@@ -0,0 +1 @@
+exec $(dirname $0)/openqa eval -m production -V 'app->gru->enqueue_limit_assets()' "$@"
```

Comment:
Does this work with white-spaces or needs it quoting like `"$(dirname "$0")/openqa"? And I suppose `"${0%/*}"` would do it as well. And don't these files lack the shebang comment?

Reply:
To be correct it should have quoting. I thought I could avoid the shebang for a single-line simple exec sh script but then the command `file` used to detect shell scripts for shellcheck corrects "text/plain", and not shell. So I added shebangs everywhere and quotes.